src/main/java/seedu/address/logic/parser/ParseArgument.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.lecture.LectureName;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.video.VideoName;
+
+
+/**
+ * A dummy class containing utility methods used for parsing strings into ModuleCode, LectureName, and Video.
+ */
+public class ParseArgument {
```

Comment:
This class seems to serve the same purpose as `seedu.address.logic.parser.ParserUtil.java`